src/Tizen.NUI/src/public/Window.cs

```diff
@@ -1310,5 +1310,52 @@ protected override void ReleaseSwigCPtr(System.Runtime.InteropServices.HandleRef
         {
             Interop.Window.delete_Window(swigCPtr);
         }
+
+        /// <summary>
+        /// FrameCallbackType
+        /// </summary>
+        /// <param name="frameId"></param>
+        [EditorBrowsable(EditorBrowsableState.Never)]
+        public delegate void FrameCallbackType(int frameId);
+
+        /// <summary>
+        /// AddFrameRenderedCallback
```

Comment:
please update the native guide. 

Adds a callback that is called when the frame rendering is done by the graphics driver.

  * @param[in] callback The function to call
   * @param[in] frameId The Id to specify the frame. It will be passed when the callback is called.
   *
   * @note A callback of the following type may be used:
   * @code
   *   void MyFunction( int frameId );
   * @endcode
   * This callback will be deleted once it is called.
   *
   * @note Ownership of the callback is passed onto this class.